timescaledb/tutorials/nfl-analytics/join-with-relational.md

```diff
@@ -0,0 +1,96 @@
+
+### The Mile-High advantage
+
+In professional football, it's a well-understood phenomenon that teams that
+visit Denver's Mile-High stadium are at a disadvantage because unlike the home team
+(the Denver Broncos), they are not accustomed to playing in high altitude.
+
+Earlier we ingested stadium data. Now we can run a query to see the performance of
+players at each position when they are playing at Mile High Stadium.
+
+Use this query to compare the average acceleration and yards run of individual players when they are performing in stadiums outside of Denver versus when they are playing in Denver. The columns `avg_acc_den` and `avg_yards_den` represent the acceleration and yard statistics while in Denver.  
+
+```sql
+WITH stat_vals AS (
+-- This table collects the summed yard and avg acceleration data of a player during one game
+  SELECT a.player_id, display_name, SUM(yards) AS yards, AVG(acc) AS acc, team, gameid 
+  FROM player_yards_by_game a
+  LEFT JOIN player p ON a.player_id = p.player_id 
+  GROUP BY a.player_id, display_name, gameid, team
+), team_data AS (
+-- This table gets us the team information so that we can filter on teams
+	SELECT a.player_id, acc, yards, a.gameid,
+		CASE 
+			WHEN a.team = 'away' THEN g.visitor_team 
+			WHEN a.team = 'home' THEN g.home_team 
+			ELSE NULL 
+			END AS team_name,
+		g.home_team
+	FROM stat_vals a
+	LEFT JOIN game g ON a.gameid = g.game_id 
+), avg_stats AS (
+-- This table takes the avg acceleration and yards run for players when they are not in denver
+-- and then when they are in denver
+SELECT p.player_id, p.display_name, 
+	AVG(acc) FILTER (WHERE team_name != 'DEN' AND home_team !='DEN') AS avg_acc, 
+	AVG(acc) FILTER (WHERE team_name != 'DEN' AND home_team = 'DEN') AS avg_acc_den,
+	AVG(yards) FILTER (WHERE team_name != 'DEN' AND home_team !='DEN') AS avg_yards, 
+	AVG(yards) FILTER (WHERE team_name != 'DEN' AND home_team = 'DEN') AS avg_yards_den, 
+	COUNT(gameid) FILTER (WHERE team_name != 'DEN' AND home_team !='DEN') AS games,
+	COUNT(gameid) FILTER (WHERE team_name != 'DEN' AND home_team ='DEN') AS games_den
+FROM team_data t
+LEFT JOIN player p ON t.player_id = p.player_id 
+GROUP BY p.player_id, p.display_name
+)
+SELECT * FROM avg_stats
+WHERE avg_acc IS NOT NULL AND avg_acc_den IS NOT NULL
+ORDER BY avg_acc DESC, avg_acc_den DESC
+```
+
+You can see that generally, it appears many players may have worse acceleration and average number of yards run per game while playing in Denver. 
+
+### Grass vs. turf, the eternal (football) question
+
+Players often say they "feel" faster on artifical turf. How much faster are they
+in reality?
+
+Using this query you will extract the average acceleration that a player has while using turf verses grass. The column `avg_acc_turf` represents the players average acceleration while using artificial turf, and `avg_acc_grass` represents their average acceleration while on grass. 
+
```

Comment:
I think we need to add a little exposition here that explains the two tables you're joining on. Like:

In this case, we will join the `stadium_info` and `team_data` tables on the `location` of the game. After that, we'll be able to extract the `surface` data from the `stadium_info` table.